ocs_ci/ocs/ui/base_ui.py

```diff
@@ -882,8 +883,18 @@ def navigate_block_pool_page(self):
 
         """
         logger.info("Navigate to block pools page")
-        self.navigate_to_ocs_operator_page()
-        self.do_click(locator=self.page_nav["block_pool_link"])
+        self.nav_odf_default_page()
```

Comment:
please use this approach, so we do not repeat code and save time on maintenance. Each page prints it's name when navigating into it

            PageNavigator()
            .nav_odf_default_page()
            .nav_storage_systems_tab()
            .nav_storagecluster_storagesystem_details()
            .nav_ceph_blockpool()

Reply:
Made the changes.